app/src/main/java/org/literacyapp/startguide/receiver/BootReceiver.java

```diff
@@ -0,0 +1,21 @@
+package org.literacyapp.startguide.receiver;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import org.literacyapp.startguide.content.swipe.SwipeUpDownActivity;
+import org.literacyapp.startguide.util.StartPrefsHelper;
+
+
+public class BootReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        if (Intent.ACTION_BOOT_COMPLETED.equals(intent.getAction()) && StartPrefsHelper.startAfterBoot(context)) {

```

Comment:
@sladomic The start guide app starts automatically on boot the first 7 days, based on UserPrefs. How do we make this work together with the authentication mechanism?